bucket/cru.json

```
@@ -0,0 +1,18 @@
+{
+    "homepage": "https://www.monitortests.com/forum/Thread-Custom-Resolution-Utility-CRU",
+    "description": "Custom Resolution Utility (CRU) allows custom resolutions to be defined by creating EDID overrides directly in the registry without dealing with .inf files.",
+    "version": "1.4.1",
+    "license": "Public|Unspecified",
+    "url": "https://www.monitortests.com/cru-1.4.1.zip",
+    "hash": "74e661d0d30203b123ef0bf9e25746f8434c1627f4e63e2338bfaff39238b783",
+    "shortcuts": [
+        [
+            "CRU.exe",
+            "Custom Resolution Utility"
+        ]
+    ],
+    "checkver": "cru-([\\w.]+).zip",
```

Why  `\w`? I do not see any special character in older versions.

```suggestion
    "checkver": "cru-([\\d.]+)\\.zip",
```